config/filesystems.php

```diff
@@ -35,6 +35,7 @@
             'root' => storage_path('app/private'),
             'serve' => true,
             'throw' => false,
+            'report' => false,
```

Comment:
Some unexpected changes here. Did you use some other starter code for this project?